### PR TITLE
created wish list page

### PR DESCRIPTION
**Pull request by Rockyraven**
created wish list page

### CODE REVIEW

File: src/component/card/ProductCard.jsx [hunk 1]
```diff
@@ -1,38 +1,79 @@
-import { useNav } from 'context/NavContext'
-import React, { useState } from 'react'
-import './productcard.css'
-export const ProductCard = ({ title, brand, discount, price, actualPrice, imgSrc, rating }) => {
+import { useNav } from "context/NavContext";
+import React, { useState } from "react";
+import "./productcard.css";
```
Comment (bishnoimukesh) [inline src/component/card/ProductCard.jsx]: avoid removing and rewriting the same code

File: src/component/card/ProductCard.jsx [hunk 2]
```diff
@@ -1,38 +1,79 @@
...
+      },
+    ]);
+  };
 
-            </div>
-        
-    )
-}
+  const removeFromWishList = () => {
+    console.log("delete");
```
Comment (bishnoimukesh) [inline src/component/card/ProductCard.jsx]: remove console before pushing the code

File: src/pages/product/Product.js [hunk 3]
```diff
@@ -20,32 +20,25 @@ export const Product = () => {
...
   });
+  
   const categoryFilteredProduct = categoryFiltered(
     product,
     state
   );
-  const brandFilteredProduct = brandFiltered(
-    product,
-    state
-  );
```
Comment (bishnoimukesh) [inline src/pages/product/Product.js:32]: avoid this

**Review by bishnoimukesh — COMMENTED** [review]
avoid that removing and writing the same code again and some changes are suggested.
rest everthing is good to move

File: src/context/WishContext.js [hunk 4]
```diff
@@ -3,13 +3,29 @@ import { createContext, useContext,  useEffect,  useState } from "react";
...
-    useEffect(()=>{
-        console.log(wishlist);
+    const [ wishlist, setWishList ] = useState(JSON.parse(localStorage.getItem ("wishlist")));
+    useEffect(()=> {
+        localStorage.setItem("wishlist", JSON.stringify(wishlist))
     },[wishlist])
+
+  // funtions for add to cart and remove card  
+
+    const addToWishList = (product) => {
```
Comment (Shubhamkr2610) [inline src/context/WishContext.js:13]: its good to give message

**Review by Shubhamkr2610 — APPROVED** [review]
Everything looks good , code is readable and easy  to understand , You can merge it

**Pull request merged**